modules/programs/todoman.nix

```
@@ -0,0 +1,62 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.todoman;
+
+  format = pkgs.formats.keyValue { };
+
+in {
+
+  meta.maintainers = [ hm.maintainers.mikilio ];
+
+  options.programs.todoman = {
+    enable = lib.mkEnableOption "todoman";
+
+    glob = mkOption {
+      type = types.str;
+      default = "*";
+      description = ''
+        The glob expansion which matches all directories relevant.
+      '';
+      example = "*/*";
+    };
+
+    config = mkOption {
```

The convention is to call this option `extraConfig`:
```suggestion
    extraConfig = mkOption {
```